misc/testService/testMethod.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "NsSkelRpcReplier.h"
+
+class TestMethodArgs {
+public:
+	int32_t a;
+	int32_t b;
+	NSSERIALIZER_PREPARE(a,b);
+};
+
+class TestMethodResult {
+public:
+	int32_t c;
+	NSSERIALIZER_PREPARE(c);
+};
+
+class TestMethod: public nanoservices::NsSkelRpcReplier<TestMethodArgs, TestMethodResult> {
+public:
+	std::shared_ptr<TestMethodResult> processRequest(std::shared_ptr<TestMethodArgs> args) throw(nanoservices::NsException);
```

Comment:
```suggestion
	std::shared_ptr<TestMethodResult> processRequest(std::shared_ptr<TestMethodArgs> args) noexcept(false);
```

Боюсь, придётся соответствовать общепринятой практике

Reply:
Ни вапрос, пройдусь везде и заменю.

Кстати auto_ptr тоже deprecated а в msgpack-c он есть. Возможно это изза директивы про Cxx03

Reply:
`noexcept(false)` в принципе указывать не надо, т.к. компилятору пофиг.

А вот `noexcept(true)` там где необходимо стоит указать

Reply:
Сделал